README.md

```diff
@@ -1,48 +1,66 @@
-# Using R with Python
+# RStudio Connect & Python
+
+RStudio Connect is a publishing platform for the work your team creates in R and Python.
+This repository contains examples of Python content you can deploy to Connect, including:
+
+## Interactive apps
+
+- [Streamlit](./streamlit-income-share/README.md)
+- [Dash](dash-app/README.MD)
+
```

Comment:
I wonder whether we should add a README.md to the Flask app [here](https://github.com/sol-eng/python-examples/tree/main/flask-sentiment-analysis-app) and add Flask to the apps section? @edavidaja

Reply:
https://github.com/sol-eng/python-examples/tree/readmes/flask-sentiment-analysis-api

I figured mentioning flask in the web apis section was sufficient but if you think it needs to be more explicit I can add it there as well.

Reply:
I was suggesting that because a customer asked about whether we restricting the functions of Flask to API only  on RSC. That's probably caused by a misunderstanding on their part to be fair.